CONTRIBUTING.md

```diff
@@ -164,29 +472,39 @@ New committers are approved by majority vote of the TSC ([see TSC charter](https
         * Use the imperative mood in the subject line
         * Wrap the body at 72 characters
         * Use the body to explain what and why vs. how
-     
-- Example commit message
-
-```
-Fix OOM caused by foo in bar
-
-Foo was pack ratting ByteBuffers causing OOM.
-
-Cherry-pick of c2bf3b0 (https://github.com/foo/bar/pull/123/commits/c2bf3b025ff456921c4c5ff4bcb814740e823061)
-
-Co-authored-by: Foo Bar <foo@bar.com>
-```
+    * Ensure all code is peer reviewed within your own organization or peers before submitting
+    * Implement and address existing feedback before requesting further review
+    * Make a good faith effort to locate example or referential code before requesting someone else direct you towards it
+    * If you see a lack of documentation, create a separate commit with draft documentation to fill the gap
+        * This documentation can be iterated on same as any code commit, demonstrate in real time that you are learning the code section
+    * Implement or modify relevant tests, otherwise provide clear explanation why test updates were not necessary
+    * Tag your PR with affected code areas as best as you can, it’s okay to tag too many, better to cut down irrelevant tags than miss getting input from relevant subject matter experts
 
 - Example PR message
-  - please refer [here](https://github.com/prestodb/presto/blob/master/pull_request_template.md)
+    - please refer [here](https://github.com/prestodb/presto/blob/master/pull_request_template.md)
```

Comment:
I feel like we could potentially remove this? It's said earlier that you should follow the PR template guidelines